default_config.json

```diff
@@ -26,5 +26,6 @@
     "environment_name" : "MiniGrid-arie-test-v0",
     "replay_buffer_module": "services.algorithms.dqn.replay_buffer.simple",
     "replay_buffer_prioritization" : 0.5,
-    "target_sync_frequency" : 50
+    "target_sync_frequency" : 50,
+      "clip_norm": 0
```

Comment:
align with others

Reply:
I tried to fix this but it just wouldn't align :/